### PR TITLE
rename keyword `function` to `fn`

### PR DESCRIPTION
**Pull request by baszalmstra**

### CODE REVIEW

File: crates/mun/test/main.mun [hunk 1]
```diff
@@ -1,19 +1,19 @@
-// Function to add two floats
-function add(a:float, b:float):float {
+// fn to add two floats
```
Comment (Wodann) [inline crates/mun/test/main.mun]: You inadvertently replaces Function in this comment too.

File: crates/mun/test/main.mun [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+fn add(a:float, b:float):float {
   let c = a+b+5.0
   c
 }
 
-// Function to subtract two floats
-function subtract(a:float, b:float):float {
+// fn to subtract two floats
```
Comment (Wodann) [inline crates/mun/test/main.mun]: Likewise

File: crates/mun/test/main.mun [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+fn subtract(a:float, b:float):float {
   a-b
 }
 
-// Function to subtract two floats
-function multiply(a:float, b:float):float {
+// fn to subtract two floats
```
Comment (Wodann) [inline crates/mun/test/main.mun]: One more

**Comment by baszalmstra** [comment]
[Rename `function` to `fn`](https://trello.com/c/WACqSHp1/39-rename-function-to-fn)

**Comment by Wodann** [comment]
It seems good. Once you remove the draft status, the PR can be merged.

**Pull request merged**